met-web/package.json

```diff
@@ -14,6 +14,7 @@
         "@mui/lab": "^5.0.0-alpha.91",
         "@mui/material": "^5.6.1",
         "@mui/x-date-pickers": "^5.0.5",
+        "@mui/x-date-pickers-pro": "^6.4.0",
```

Comment:
what does the pro do?
what licence it has?

Reply:
Pro is another material ui sub library, there's no licensing issue here though the issue is only when we use the DateRangeCalendar component.

Reply:
the pro component is just cleaner and does not require the render input prop

Reply:
can you point to some links on the licensing? also pro vs pickers in terms of the 'pro' term?

Reply:
https://mui.com/pricing/ this is the licensing chart 

the pro library picker:
`   <DatePicker
                                    value={fromDate}
                                        onChange={(newDate: Dayjs | null) => setFromDate(newDate)}
                                    />`
 The non pro picker:
 `     <DatePicker
                                    inputFormat="MM/DD/YYYY"
                              value={fromDate}
                                        onChange={(newDate: Dayjs | null) => setFromDate(newDate)}
                                    renderInput={(params) => <TextField {...params} />}
                                />`

Reply:
so are we using a non-free version? 

@jadmsaadaot thoughts here?

Reply:
no this is still the free version. It's only certain components in the library that are not free.

Reply:
Yea that sounds good ill revert to the regular date picker then and remove the pro library